service/resource/cloudformation/main_stack.go

```diff
@@ -4,6 +4,7 @@ import (
 	"bytes"
 	"text/template"
 
+	awsCF "github.com/aws/aws-sdk-go/service/cloudformation"
```

Comment:
Please change the package alias here too.

Reply:
Done.